activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-client/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>activiti-cloud-services-runtime-bundle</artifactId>
+    <groupId>org.activiti.cloud</groupId>
+    <version>7.1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>activiti-cloud-services-rest-client</artifactId>
```

Comment:
I think it's better to include `runtime-bundle` to the name of this module, so we know which service it's referencing while adding it as a dependency: `activiti-cloud-runtime-bundle-rest-client` or similar.
Don't forget to rename the folder accordingly.